src/AccessibilityInsights.Extensions/Container.cs

```diff
@@ -145,6 +145,16 @@ public static Container GetDefaultInstance()
 
             return _defaultInstance;
         }
+
+        /// <summary>
+        /// Set the ReleaseChannel for update
+        /// </summary>
+        /// <param name="releaseChannel">The value to use in the AutoUpdate code</param>
+        public static void SetAutoUpdateReleaseChannel(string releaseChannel)
```

Comment:
>string [](start = 55, length = 6)

This uses a string to keep Extensions definitions decoupled from the SetupLibrary extension